src/Symfony/Component/Validator/composer.json

```diff
@@ -26,7 +26,7 @@
         "symfony/http-foundation": "~4.1",
         "symfony/http-kernel": "~3.4|~4.0",
         "symfony/var-dumper": "~3.4|~4.0",
-        "symfony/intl": "~4.1",
+        "symfony/intl": "^4.3",
```

Comment:
We need to update the conflict rule to forbid projects from using older versions.